src/main/java/com/diamondfire/helpbot/bot/command/impl/other/mod/PurgeCommand.java

```diff
@@ -51,6 +55,28 @@ public void run(CommandEvent event) {
         } else {
             TextChannel channel = event.getChannel();
             channel.getHistory().retrievePast(messagesToRemove).queue((messages) -> {
+                // Adds the messages to the messageBuilder object
+                MessageBuilder messageBuilder = new MessageBuilder();
+                messageBuilder.append("Here are the messages you purged;\n");
+                
+                // Iterates through the message history and appends the values to the MessageBuilder.
+                for (Message m : messages) {
+                    messageBuilder
+                            .append("[")
+                            .append(m.getTimeCreated().format(DateTimeFormatter.RFC_1123_DATE_TIME))
```

Comment:
Wouldn't it be cleaner to use String#format instead of a bunch of chained append calls?